### PR TITLE
Refactor/add new fonts

### PR DESCRIPTION
**Pull request by W3stside**
This PR is one of many to be part of the design refactor issue #301 

Changes:

1. Adds new fonts "Averta" as outlined in design spec
2. Moves styles to appropriate folder
3. creation of `assets` folder for fonts and images

### CODE REVIEW

File: webpack.config.babel.js [hunk 1]
```diff
@@ -54,6 +54,16 @@ module.exports = ({ stats = false } = {}) => ({
         test: /\.css$/,
         use: ['style-loader', 'css-loader'],
       },
+      {
+        test: /\.(ttf|otf|eot|woff2?)(\?[a-z0-9]+)?$/,
+        use: {
+          loader: 'file-loader',
+          options: {
+            name: '[name].[ext]',
+            mimetype: 'application/font-otf',
```
Comment (Velenir) [inline webpack.config.babel.js]: `mimetype` is for `url-loader`, file-loader doesn't care. Not to mention different fonts which all can be matched by the test regexp need different mimetypes. Also mimetype in url-loader is determined from file extension by default

File: .babelrc [hunk 2]
```diff
@@ -7,6 +7,7 @@
     [
       "babel-plugin-styled-components",
       {
+        "displayName": true,
```
Comment (Velenir) [inline .babelrc]: I'm pretty sure it's already true by default
Try with false and compare.

File: src/styles/fonts.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { css } from 'styled-components'
+
+import AvertaRegular from '../assets/fonts/Averta-Regular.otf'
+import AvertaBold from '../assets/fonts/Averta-Bold.otf'
+import AvertaExtraBold from '../assets/fonts/Averta-Extrabold.otf'
+
+const fontFace = css`
+  @font-face {
+    font-family: 'Averta';
+    src: url(${AvertaRegular});
```
Comment (Velenir) [inline src/styles/fonts.ts]:
```suggestion
    src: url(${AvertaRegular}) format('opentype');
```
for each .otf font

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr312--dexreact.review.gnosisdev.com

**Comment by Velenir** [comment]
> Coo, just one question, is the default font (while the network font is not being loaded) a bit similar to this Averta?

Depends on your system
pr312--dexreact review gnosisdev com_trade_DAI-USDC__sell=0 buy=0 (1)
pr312--dexreact review gnosisdev com_trade_DAI-USDC__sell=0 buy=0 (2)

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr312--dexreact.review.gnosisdev.com

**Pull request merged**